tests/test_mockgun.py

```diff
@@ -269,6 +269,19 @@ def test_find_by_sub_entity_field(self):
         )
         self.assertEqual(len(items), 1)
 
+    def test_find_with_none(self):
+        """
+        Ensures comparison with multi-entity fields and None works.
+        """
+        items = self._mockgun.find("PipelineConfiguration", [["users", "is", None]], ["users"])
+        self.assertEqual(len(items), 1)
```

Comment:
Minor: By convention, asserEqual first argument is the expected value, the second would be the value that is unknown.